akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/VoiceApp.scala

```diff
@@ -768,6 +771,31 @@ object VoiceApp extends SystemConfiguration {
       }
   }
 
+  def setListenOnlyInputInVoiceConf(
+    liveMeeting:              LiveMeeting,
+    outGW:                    OutMsgRouter,
+    userId:                   String,
+    listenOnlyInputDevice:    Boolean
+  )(implicit context: ActorContext): Unit = {

```

Comment:
This function doesn't require a context.

```suggestion
  ): Unit = {
```